### PR TITLE
[feat] Database Option

### PR DESCRIPTION
**Pull request by luchiago**
Feature requested in the #1467. I added a new option in `print_settings.py` to show the current database. But we need to approve and merge the #1553 PR, because I made some refactoring in this file through this PR

### CODE REVIEW

File: django_extensions/management/commands/print_settings.py [hunk 1]
```diff
@@ -26,34 +29,37 @@ def add_arguments(self, parser):
...
-            default=4,
+            default=DEFAULT_INDENT,
             dest='indent',
             type=int,
             help='Specifies indent level for JSON and YAML'
         )
+        parser.add_argument(
+            '--show-secrets',
+            default=DEFAULT_SECRETS,
+            dest='show secrets',
```
Comment (trbs) [inline django_extensions/management/commands/print_settings.py]: please do not use spaces, use underscores.

File: django_extensions/management/commands/print_settings.py [hunk 2]
```diff
@@ -111,3 +111,13 @@ def include_attr(attr, settings):
     def print_simple(a_dict):
         for key, value in a_dict.items():
             print("%-40s = %r" % (key, value))
+
+    @staticmethod
+    def safe_settings():
+        try:
+            return SafeExceptionReporterFilter().get_safe_settings()
```
Comment (trbs) [inline django_extensions/management/commands/print_settings.py:131]: If django < 3.0 does not have this class then the import above will already have failed and this code is meaningless.

Either the comment below is wrong or we should catch `ImportError` during importing.

FYI: This entire code can be replaced to function at startup time which seem to be preferred to this way if it served no additional function.

File: django_extensions/management/commands/print_settings.py [hunk 3]
```diff
@@ -47,11 +47,17 @@ def add_arguments(self, parser):
             type=bool,
             help="Specifies if should be revealed the value of secrets",
         )
+        parser.add_argument(
+            "--database",
+            dest="show database",
```
Comment (trbs) [inline django_extensions/management/commands/print_settings.py:52]: no spaces, just underscores :-)

**Comment by trbs** [comment]
I guess I don't understand what purpose this PR serves... `DATABASES` is already included on the output of `print_settings` ?

**Comment by luchiago** [comment]
> I guess I don't understand what purpose this PR serves... `DATABASES` is already included on the output of `print_settings` ?

@trbs As I said on #1467, I misunderstand the purpose. Let's close this PR and I can do the right one.

**Pull request closed without merging**